splitgraph/commandline/cloud.py

```diff
@@ -1310,6 +1310,29 @@ def tunnel_c(remote: str, local_address: str, namespace_and_repository: Optional
         start_repository_tunnel(remote, local_address, namespace_and_repository)
 
 
+@click.command("sync-to-seafowl")
+@click.option("--remote", default="data.splitgraph.com", help="Name of the remote registry to use.")
+@click.argument("vdb_id", type=str)
+@click.argument("seafowl_hostname", type=str)
+@click.argument("seafowl_secret", type=str)
+def sync_to_seafowl_c(remote: str, vdb_id: str, seafowl_hostname: str, seafowl_secret: str):
+    """
+    Sync the tables within a Splitgraph VDB to a Seafowl instance.
+
+    """
+
+    from splitgraph.cloud import GQLAPIClient
+
+    client = GQLAPIClient(remote)
+    task_id = client.start_seafowl_sync(vdb_id, seafowl_hostname, seafowl_secret)
+    final_status = wait_for_job(task_id, lambda: client.get_export_job_status(task_id))
+    if final_status.status == "SUCCESS":
+        assert final_status.output
```

Comment:
Does the Seafowl sync job actually return anything?

Reply:
It displays the URL for the parquet export of the tables. That's not something we really need, but a list of successfully synced tables might be useful. I updated the PR to display this on success.